cmd/server/download.go

```diff
@@ -62,6 +62,7 @@ type DownloadStats struct {
 	BISEntities       int `json:"bisEntities"`
 	MilitaryEndUsers  int `json:"militaryEndUsers"`
 	SectoralSanctions int `json:"sectoralSanctions"`
+	Unverified        int `json:"Unverified"`
```

Comment:
Just to be clear where these entries came from. 

```suggestion
	Unverified        int `json:"unverifiedCSL"`
```